tests/Doctrine/Tests/ORM/Functional/PaginationTest.php

```diff
@@ -179,25 +336,44 @@ public function testCountQueryStripsParametersInSelect()
         // if select part of query is replaced with count(...) paginator should remove
         // parameters from query object not used in new query.
         $this->assertCount(1, $getCountQuery->invoke($paginator)->getParameters());
-        $this->assertCount(3, $paginator);
+        $this->assertCount(9, $paginator);
     }
 
     public function populate()
     {
-        for ($i = 0; $i < 3; $i++) {
+        $groups = [];
```

Comment:
please don't use the short array syntax. Making the test compatible with PHP 5.3 would allow us to backport it to the 2.4 branch to be able to fix the pagination bugs in it too and have tests for the fixes